src/main/java/uk/gov/hmcts/divorce/systemupdate/schedule/conditionalorder/SystemRemindApplicantsApplyForCOrderTask.java

```diff
@@ -87,11 +109,43 @@ public void run() {
     }
 
     private void remindJointApplicants(CaseDetails caseDetails, User user, String serviceAuth) {
+
+        CaseData caseData = objectMapper.convertValue(caseDetails.getData(), CaseData.class);
+        String state = caseDetails.getState();
+
+        try {
+            if (AwaitingConditionalOrder.name().equals(state) || ConditionalOrderDrafted.name().equals(state)) {
+                notificationDispatcher.send(awaitingConditionalOrderReminderNotification, caseData, caseDetails.getId());
+            } else {
+                notificationDispatcher.send(conditionalOrderPendingReminderNotification, caseData, caseDetails.getId());
+            }
+        } catch (NotificationException | HttpServerErrorException exception) {
+            log.error("Notification for SystemRemindApplicantsApplyForCOrderTask has failed with exception {} for case id {}",
+                exception.getMessage(), caseDetails.getId());
+
+            if (caseData.getConditionalOrder().getCronRetriesRemindApplicantApplyCo() == null) {
+                caseData.getConditionalOrder().setCronRetriesRemindApplicantApplyCo(0);
+            }
+
+            if (caseData.getConditionalOrder().getCronRetriesRemindApplicantApplyCo() < MAX_RETRIES) {
+                caseData.getConditionalOrder().setCronRetriesRemindApplicantApplyCo(
+                    caseData.getConditionalOrder().getCronRetriesRemindApplicantApplyCo() + 1);
+
+                caseDetails.setData(objectMapper.convertValue(caseData, new TypeReference<>() {}));
+                ccdUpdateService.submitEvent(caseDetails, SYSTEM_UPDATE_CASE, user, serviceAuth);
+            }
+
+            // return as we don't want to continue and call the 'remind applicant' event.
+            return;
+        }
+
+        // If notifications are successful, continue to call the system remind applications CO event.
         try {
             log.info(
                 "20Week holding period +14days elapsed for Case({}) - reminding Joint Applicants they can apply for a Conditional Order",
                 caseDetails.getId()
             );
+            caseDetails.setData(objectMapper.convertValue(caseData, new TypeReference<>() {}));
             ccdUpdateService.submitEvent(caseDetails, SYSTEM_REMIND_APPLICANTS_CONDITIONAL_ORDER, user, serviceAuth);
 
         } catch (final CcdManagementException e) {
```

Comment:
I think this part of the code can be another method, and then added inside the earlier try catch so that there will be no need for the return statement on line 139